tests/native_binary/assertenv.cc

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+#include <string.h>
+
+#ifdef _WIN32
+#define OS_VAR "APPDATA"
+#define strncasecmp _strnicmp
+#else
+#define OS_VAR "HOME"
+#endif
+
+enum vars_to_be_found {
```

Comment:
Could you implement this test in a less hacky way?

For example, get rid of the enum, and just search for 3 strings. Use set found to true/false.